xarray/core/accessor_str.py

```diff
@@ -672,6 +672,23 @@ def capitalize(self) -> T_DataArray:
         Returns
         -------
         capitalized : same type as values
+
+        Example
+        -------
```

Comment:
```suggestion
        Examples
        --------
```

Forgot to mention earlier, the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#examples) format, which we use, recommends "Examples"

Reply:
> Forgot to mention earlier, the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#examples) format, which we use, recommends "Examples"

I'll update it, thanks!